engine/src/main/java/com/redhat/cloud/notifications/exports/TransformationException.java

```diff
@@ -0,0 +1,10 @@
+package com.redhat.cloud.notifications.exports;
+
+/**
+ * A wrapper exception class which wraps any transformation errors.
+ */
+public class TransformationException extends Exception {
```

Comment:
This is not used in this PR. Will it be used later?

Reply:
I was thinking to standardize what the transformers should look like. I also modified it in order to show that it might throw an exception, which is basically a generic wrapper to the multiple exceptions the transformers might throw. That way we have to handle just a single exception... However I am not very much convinced about it.

I also thought that this might be the way to end up building a factory for the transformers, but I don't know...